var/spack/repos/builtin/packages/py-pennylane-lightning-kokkos/package.py

```diff
@@ -42,14 +43,14 @@ class PyPennylaneLightningKokkos(CMakePackage, PythonExtension, CudaPackage, ROC
 
     # CUDA
     for val in CudaPackage.cuda_arch_values:
-        depends_on("kokkos@:3.7.01 cuda_arch={0}".format(val), when="cuda_arch={0}".format(val))
+        depends_on("kokkos@:4.0.01 cuda_arch={0}".format(val), when="cuda_arch={0}".format(val))
```

Comment:
Is this true for all versions or just for 0.31?

Reply:
It appears it doesn't, will fix this. Thanks @adamjstewart .